OpenTok/OpenTok.cs

```diff
@@ -8,47 +8,51 @@
 
 namespace OpenTokSDK
 {
-    /**
-    * Contains methods for creating OpenTok sessions, generating tokens, and working with archives.
-    * <p>
-    * To create a new OpenTok object, call the OpenTok() constructor with your OpenTok API key
-    * and the API secret for your <a href="https://tokbox.com/account">OpenTok project</a>.
-    * Do not publicly share your API secret. You will use it with the OpenTok constructor
-    * (only on your web server) to create OpenTok sessions.
-    */
+    /// <summary>
+    /// Contains methods for creating OpenTok sessions, generating tokens, and working with archives.
+    /// <para>
+    /// To create a new OpenTok object, call the OpenTok() constructor with your OpenTok API key
+    /// and the API secret for your <see href="https://tokbox.com/account">OpenTok project</see>.
```

Comment:
These external links do not work (at least with Doxygen generating the docs). Can you please change all `see` links to external URLs to use `a` tags (throughout)?

```suggestion
    /// and the API secret for your <a href="https://tokbox.com/account">OpenTok project</a>.
```

Reply:
Did you confirm that `a` works? `<see href="">` appears to be the official syntax (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/see#remarks) but it looks like this is not supported by Doxygen (http://doxygen.nl/manual/xmlcmds.html).

Reply:
It looks like Visual Studio supports both - @jeffswartz - does doxygen support this as well?

Reply:
Yes, the `a` tag works in Doxygen. And it does not support the `<see href >` syntax.

Reply:
Thanks @jeffswartz. Will update the links later today.